src/components/panel/panel.js

```diff
@@ -832,6 +850,47 @@ MdPanelRef.prototype._addStyles = function() {
 };
 
 
+/**
+ * Add ARIA-specific attributes appropriate for panels.
+ */
+MdPanelRef.prototype._configureAria = function() {
```

Comment:
I was matching mdDialog since we will likely have more content here later, but I am fine with removing it for now.